[KOGITO-5300] Updating documentation of addons modules (#1463)
kiegroup/kogito-runtimes: addons/common/cloudevents/README.md

# Kogito CloudEvents Addon

> Use this addon to make the events produced and consumed by a Kogito project accept

Reviewer: Decide whether to use "addon" or "add-on", throughout the entire document, for consistency.

> Use this addon to make the events produced and consumed by a Kogito project accept

Reviewer 2: ```suggestion
Use this add-on to make the events produced and consumed by a Kogito project accept
```

> the [CloudEvents](https://cloudevents.io) format. There are many use cases where you can use this addon.

Reviewer 2: ```suggestion
the [CloudEvents](https://cloudevents.io) format. There are many use cases where you can use this add-on.
```

> the [CloudEvents](https://cloudevents.io) format. There are many use cases where you will need to use this addon. Please

Reviewer: ```suggestion
the [CloudEvents](https://cloudevents.io) format. There are many use cases where you can use this addon.
```

> check the following sections in our documentation to find out more:

Reviewer: ```suggestion
See the following sections in our documentation to find out more:
```
